CHANGES.md

```diff
@@ -1,3 +1,6 @@
+# 0.0.5 (2019-10-06)
```

Comment:
Thinking about it, there should be no version number and date here as this change is still unreleased.